src/extension.ts

```diff
@@ -199,12 +199,81 @@ export async function activate(ctx: vscode.ExtensionContext) {
           break;
       }
     }
+  }
 
-    if (configureOnOpen === true) {
-      runFirstTask("reconfigure");
+  const downloadLanguageServerKey = "downloadLanguageServer";
```

Comment:
Can you create an enum for the settings instead of having these strings located throughout the code?

Reply:
Do you mean like this: https://github.com/mesonbuild/vscode-meson/pull/123/commits/dbb15f11b248687894976efa299ba23904775c84#diff-c54113cf61ec99691748a3890bfbeb00e10efb3f0a76f03a0fd9ec49072e410aR133